src/ontology/mondo-edit.obo

```diff
@@ -522293,8 +522283,15 @@ name: intracranial meningioma
 def: "A meningioma that arises within the cranial cavity." [DOID:0080842]
 subset: inferred_rare
 subset: rare
-xref: DOID:0080842 {source="MONDO:equivalentTo"}
-is_a: MONDO:0016642 {source="DOID:0080842"} ! meningioma
+synonym: "brain meningioma" EXACT [MONDO:0000642]
+synonym: "brain meningioma (disease)" EXACT [MONDO:patterns/location]
+synonym: "meningioma (disease) of brain" EXACT []
+xref: DOID:0060106 {source="MONDO:equivalentTo"}
+xref: DOID:0080842 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
+is_a: MONDO:0016642 {source="DOID:0060106", source="DOID:0080842", source="MONDO:Redundant"} ! meningioma
+intersection_of: MONDO:0016642 ! meningioma
+intersection_of: disease_has_location UBERON:0000955 ! brain
```

Comment:
@nicolevasilevsky Is this correct? I think it would be more correct to have the location in the meninges.

Reply:
this came from the obsoleted term. I agree this is wrong:
https://www.mayoclinic.org/diseases-conditions/meningioma/symptoms-causes/syc-20355643

I will delete it